GarupaPico/GarupaPico/TabbedEventTopPage.xaml.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+using Xamarin.Forms;
+using Xamarin.Forms.Xaml;
+
+namespace GarupaPico
```

Comment:
Suggested Tabbed Page contents to be put under `GarupaPico.View` namespace to prevent flooding of the root folder.